source/ConditionSet.h

```diff
@@ -28,148 +28,155 @@ class DataWriter;
 
 
 // A condition set is a collection of operations on the player's set of named "conditions"; "test" operations that just
-// check the values of those conditions, and in the future "evaluation" operations that can calculate an int64_t value
-// based on the conditions.
-// Operations that can be "applied" to change the values were historically also here, but are being moved to
-// ConditionAssignments.
+// check the values of those conditions and "evaluation" operations that can calculate an int64_t value based on the
+// conditions.
 class ConditionSet {
 public:
 	ConditionSet() = default;
+	ConditionSet(const ConditionSet &) = default;
 
 	// Construct and Load() at the same time.
 	explicit ConditionSet(const DataNode &node);
 
+	// Construct a terminal with a literal value.
+	explicit ConditionSet(int64_t newLiteral);
+
 	// Load a set of conditions from the children of this node. Prints a
-	// warning if an and/or node contains assignment expressions.
+	// warning if the conditions cannot be parsed from the node.
 	void Load(const DataNode &node);
 	// Save a set of conditions.
 	void Save(DataWriter &out) const;
+	void SaveChild(int childNr, DataWriter &out) const;
+	void SaveSubset(DataWriter &out) const;
 
 	// Change this condition to always be false.
 	void MakeNever();
 
 	// Check if there are any entries in this set.
 	bool IsEmpty() const;
 
-	// Check if the given condition values satisfy this set of expressions. First applies
-	// all assignment expressions to create any temporary conditions, then evaluates.
+	// Check if this conditionSet contains valid data.
```

Comment:
```suggestion
	// Check if this condition set contains valid data.
```
Either both caps or no caps, I think.